docs/source/notebooks/pcca.ipynb

```diff
@@ -96,29 +177,136 @@
     "pcca.coarse_grained_transition_matrix"
    ]
   },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "The membership probabilities can be accessed via `pcca.memberships`, where each column corresponds to a metastable set defining a probability distribution over the microstates:"
+   ]
+  },
   {
    "cell_type": "code",
    "execution_count": null,
    "metadata": {},
    "outputs": [],
    "source": [
-    "crisp_memberships = pcca.memberships.argmax(axis=1)\n",
+    "print(f\"Memberships: {pcca.memberships.shape}\")"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "fig, axes = plt.subplots(1, 2, figsize=(15, 10))\n",
+    "\n",
+    "for i in range(len(axes)):\n",
+    "    ax = axes[i]\n",
+    "    ax.set_title(f\"Metastable set {i+1} assignment probabilities\")\n",
+    "    \n",
+    "    ax.scatter(*data.T, c=pcca.memberships[assignments, i], cmap=plt.cm.Blues)\n",
+    "norm = mpl.colors.Normalize(vmin=0, vmax=1)\n",
+    "fig.colorbar(plt.cm.ScalarMappable(norm=norm, cmap=plt.cm.Blues), ax=axes, shrink=.8);"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "As expected, the assignments nicely disentangle the two ellipsoids.\n",
```

Comment:
this is nice and the plot/colorbars are good, but the incorrect assignments at the edges would be a problem in real analysis, right?

Reply:
this is an artifact of the clustering and the actual effect in terms of transition rates is minimal. that being said i can throw more cluster centers at it and resolve that. which clutters the graph plot :)

Reply:
you don't have to! i don't think that's crucial

Reply:
if you had agglomerative clustering, that would give you better clusters here ;)

Reply:
true, true! Since I don't want to explain yet another clustering technique I opted for more centers.